builder/package.ubuntu-1804

```diff
@@ -0,0 +1,50 @@
+if [[ ! -d /tmp/output/ubuntu-1804 ]]; then
+  mkdir /tmp/output/ubuntu-1804
+fi
+
+fpm \
```

Comment:
I noticed that the deb package had a priority of extra, which is the fpm default, but also deprecated apparently. Should we set it to "optional"? https://github.com/jordansissel/fpm/issues/1398

Reply:
✅ Done - good catch